src/components/mdIcon/mdIcon.scss

```diff
@@ -10,6 +10,10 @@ $icon-size: 24px;
   font-size: $size;
 }
 
+.md-icon-font {
```

Comment:
This one was because i had trouble on font awesome. 

Just now, comparing with ionicons, i realize that is a "bug" on the font awesome side. They wire the font size at the font declaration.

Solve the proper default icon size for font awesome shouldn't be vue-material problem, therefore let it as it was originaly :+1: .